front/app/components/PostShowComponents/Comments/PublicComments.tsx

```diff
@@ -0,0 +1,194 @@
+// libraries
```

Comment:
I just moved comments here and put tabbed navigation at the index page. I didn't go into a full improvement save for a few very obvious things